app/components/ui/buttons.tsx

```diff
@@ -0,0 +1,44 @@
+import type { SerializeFrom } from '@remix-run/node';
+import { useRouteLoaderData } from '@remix-run/react';
```

Comment:
Great usage of the useRouteLoaderData hook from Remix that allows us to access data from the current route and its nested routes by using ID.